pyccel/codegen/printing/ccode.py

```diff
@@ -2911,6 +2911,12 @@ def _print_SetDiscard(self, expr):
         arg_val = self._print(expr.args[0])
         return f'{var_type}_erase({set_var}, {arg_val});\n'
 
+    def _print_SetIsDisjoint(self, expr):
+        var_type = self.get_c_type(expr.set_variable.class_type)
+        set_var = self._print(ObjectAddress(expr.set_variable))
+        arg_val = self._print(ObjectAddress(expr.args[0]))
+        return f'{var_type}_is_disjoint({set_var}, {arg_val});\n'
```

Comment:
I assume that this function comes from STC. Where is it defined exactly?

Reply:
It does not come from STC. It is defined here: `pyccel/stdlib/STC_Extensions/Set_extensions.h`